devices/Lp3943/samples/Program.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.Lp3943;
+using Iot.Device.Lp3943.Samples;
+
+I2cConnectionSettings settings = new(3, Lp3943.DefaultI2cAddress);
```

Comment:
please check other bindings with I2C samples and copy/paste the block for ESP32, it's a commented block for setting up the pins. Also add it to the read me (same, just copy/paste)